fdbserver/DeltaTree.h

```diff
@@ -1651,7 +1654,13 @@ struct DeltaTree2 {
 		} else {
 			nodeBytesUsed = 0;
 		}
+
+		ASSERT(size() <= spaceAvailable);
 		nodeBytesFree = spaceAvailable - size();
+#ifdef VALGRIND
+		// Mark unused available space as defined
+		VALGRIND_MAKE_MEM_DEFINED((uint8_t*)this + size(), nodeBytesFree);
```

Comment:
a nit: may be we can extend VALGRIND_MAKE_MEM_DEFINED MACRO to be activated iff #ifdef VALGRIND is defined. Would avoid #ifdef all over the code..

Reply:
Yes, I actually already did that in this PR but forgot to remove these, see the changes in `actorcompiler.h`.  It is done there because `#if` cannot be used within `ACTOR`s so the empty definition is the only option.